IPython/html/notebookapp.py

```diff
@@ -832,6 +833,8 @@ def init_webapp(self):
                 ssl_options['keyfile'] = self.keyfile
         else:
             ssl_options = None
+        # Disable SSLv3, since its use is discouraged.
+        ssl_options['ssl_version']=ssl.PROTOCOL_TLSv1
```

Comment:
This will fail when ssl_options is None. It should be moved into the `if` block above.